python/lsst/pipe/tasks/matchFakes.py

```diff
@@ -31,7 +31,9 @@
 
 __all__ = ["MatchFakesTask",
            "MatchFakesConfig",
-           "MatchFakesConnections"]
+           "MatchFakesConnections",
+           "MatchVariableFakesConfig",
+           "MatchVariableFakesTask"]
```

Comment:
Is there a reason to include `MatchFakesConnections` here but not `MatchVariableFakesConnections`?

Reply:
Not really, I'll remove the previous Connections.